instrumentation/rack/example/trace_demonstration3.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'rubygems'
+require 'bundler/setup'
+
+Bundler.require
+
+ENV['OTEL_TRACES_EXPORTER'] = 'console'
+OpenTelemetry::SDK.configure do |c|
+  # Uses the experimental ResponseTextMapPropagator option
```

Comment:
Would personally move this comment to line 7 and rephrase as "This uses the response_propagators option. `ResponseTextMapPropagator` is from the `opentelemetry-sdk-experimental` gem," but it's not a blocker.